javascript/javascript1/week4/character_frequency/character.js

```diff
@@ -0,0 +1,20 @@
+const characters = [];
+function getCharacterFrequencies(inputString) {
+    for (var i = 0; i < inputString.length; i++) {
+        let freq = new Object();
+        let char = inputString.charAt(i);
```

Comment:
`inputString[i]` should work too, and is a more typical way to work with strings.